tests/main/java/CalendarTest.java

```diff
@@ -0,0 +1,59 @@
+package main.java;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CalendarTest {
+    List<LocalDateTime> timesToIgnore;
+    LocalDateTime startDate;
+    LocalTime endTime;
+    Duration taskDuration;
+    Task task;
+    Event event;
```

Comment:
only need `timesToIgnore`, `calendar`, and `taskDuration`. You can add the others back if you end up making tests for them.